views/settings.php

```diff
@@ -44,7 +44,12 @@
 					<div class="step-wrap">
 						<div class="step import-step">
 							<h4 class="header"><?php esc_html_e( 'Get started with a free template.', 'wp-maintenance-mode' ); ?></h4>
-							<p class="description"><?php esc_html_e( 'Just click on one of the pre-designed templates below to get started. You can always edit and customize later, so these are a perfect starting point!', 'wp-maintenance-mode' ); ?></p>
+							<p class="description"><?php esc_html_e( 'Choose the type of template you want, powered by Otter Blocks. You can always customise your layout.', 'wp-maintenance-mode' ); ?></p>
+							<?php
+							if ( ! is_plugin_active( 'otter-blocks/otter-blocks.php' ) ) {
```

Comment:
It's not the best way to detect as someone might have Otter with a different folder name, you can also maybe check this *OR* `OTTER_BLOCKS_VERSION` constant.

Reply:
After that you can put it for QA.